### ISSUE TITLE
I got some errors after upgrading Xcode 8 and Swift 3

### ISSUE THREAD

**Issue by timemake** [simonng/YouTube-API-Demo#4]
The code was working fine for Swift 2 but I got "Ambiguous reference to member 'subscript' errors on Swift 3.

let items: Array<Dictionary<NSObject, AnyObject>> = resultsDict["items"] as! Array<Dictionary<NSObject, AnyObject>>

let playlistSnippetDict = items[i]["snippet"] as! Dictionary<NSObject, AnyObject>

videoTitle.text = videoDetails["title"] as? String

videoThumbnail.image = UIImage(data: try! Data(contentsOf: URL(string: (videoDetails["thumbnail"] as? String)!)!))

viewController.videoID = videosArray[selectedVideoIndex]["videoID"] as! String

**Comment by iosteam2017** [comment]
Did you fix it? I have the same issues, would appreciate your help)

**Comment by timemake** [comment]
Nope, it seems like there is no update in this group so I decided to build without using YT-API again.

**Comment by dhaneshgosai** [comment]
Any Update on Swift 3.0 warning and error I am getting same error with swift 3.0. and while I try to resolved it its not working as expected.
